### PR TITLE
Place lkaccess imports at the top of the function calls

### PR DESCRIPTION
**Pull request by gregjohnso**
trying to make it so we can get automagical tests that work

### CODE REVIEW

**Comment by jcass11** [comment]
It doesn't look like any reviews are assigned to this PR. Did you mean to request me and have me merge it or was this intentional?

File: fov_processing_pipeline/stats/stats.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import numpy as np
 import matplotlib.pyplot as plt
-import warnings
```
Comment (AetherUnbound) [inline fov_processing_pipeline/stats/stats.py]: I assume this was just an unused import?

File: setup.py [hunk 2]
```diff
@@ -35,20 +35,21 @@
...
+    "lkaccess",
 ]
 
 interactive_requirements = [
     "altair",
     "jupyterlab",
     "matplotlib",
-    "ipykernel"
+    "ipykernel",
+    "lkaccess",
```
Comment (AetherUnbound) [inline setup.py]: Should this be in interactive requirements? If you wanted to install multiple groups you could do `pip install fov_processing_pipeline[dev,interactive]`

File: fov_processing_pipeline/data.py [hunk 3]
```diff
@@ -10,6 +8,8 @@
 
 
 def get_cell_data():
+    import lkaccess
```
Comment (AetherUnbound) [inline fov_processing_pipeline/data.py]: I would just add a comment or TODO here to the effect of `Move this to top-level imports if/when lkaccess becomes open source`

**Pull request merged**